### PR TITLE
java Generics task

### PR DESCRIPTION
**Pull request by NataliaBodrug**
Hello all,
Please review java Generics task.
Thank you!

### CODE REVIEW

File: src/test/java/javaGenerics/League.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package javaGenerics;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class League<T extends Team> {
+
+    String name;
```
Comment (FroggyFeet) [inline src/test/java/javaGenerics/League.java:9]: access modifiers, here and below

File: src/test/java/javaGenerics/Main.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
+package javaGenerics;
```
Comment (FroggyFeet) [inline src/test/java/javaGenerics/Main.java:1]: No CamelCase in package names

Comment (NataliaBodrug) [reply src/test/java/javaGenerics/Main.java:1]: actually, after discussion in first topic with Ana-Maria and Roman, was agreed to write this way(like camel Case),  please confirm if "javagenerics" will be better, and if i have to change for all packages?

File: src/test/java/javaGenerics/League.java (hunk 1 above)
Comment (NataliaBodrug) [reply src/test/java/javaGenerics/League.java:9]: done

File: src/test/java/javaGenerics/Main.java (hunk 2 above)
Comment (NataliaBodrug) [reply src/test/java/javaGenerics/Main.java]: changed to "generics.nata"

**Pull request merged**